data-es7.js

```diff
@@ -773,7 +801,9 @@ exports.tests = [
   category: 'strawman',
   link: 'https://github.com/DavidBruant/Map-Set.prototype.toJSON',
   exec: function(){/*
-    return JSON.stringify(new Set([1, 2, 3, 2, 1])) === '[1,2,3]';
+    var set = new Set();
+    [1, 2, 3, 2, 1].forEach(function (i) { set.add(i); });
+    return JSON.stringify(set) === '[1,2,3]';
```

Comment:
To be honest, that's up to the `JSON.stringify` implementation calling `toJSON` - the responsibility of ` toJSON` method is solely to return something that `JSON.stringify` can serialize. I could definitely make a set that contains a set, but I don't think that would actually test much.